hibernate-core/src/main/java/org/hibernate/engine/spi/ActionQueue.java

```diff
@@ -228,6 +228,10 @@ public ActionQueue(SessionImplementor session) {
 		isTransactionCoordinatorShared = false;
 	}
 
+	public ExecutableList<AbstractEntityInsertAction> getInsertions() {
+		return insertions;
+	}
+
```

Comment:
Backward compatibility is a concern in Hibernate codebase evolution and we would be cautious to change its public API (like adding a new public method) for other Hibernate projects or jboss product might be broken. So approach without breaking backward compatibility has a pros accordingly.

Reply:
Sure it's not a good idea, but I need it here, I think it's safe enough since adding new getter doesn't break backward compatibility normally.